install_config/aggregate_logging.adoc

```diff
@@ -346,8 +356,14 @@ spec:
       nodeSelector:
         nodelabel: logging-es-node-1
 ----
+
+Alternatively you can use the `$oc patch` command
+----
+$ oc patch dc/logging-es-{unique name} -p '{"spec":{"template":{"spec":{"nodeSelector":{"nodeLabel":"logging-es-node-1"}}}}}'
```

Comment:
For readability (even if it does scroll):

```
$ oc patch dc/logging-es-{unique name} \
   -p '{"spec":{"template":{"spec":{"nodeSelector":{"nodeLabel":"logging-es-node-1"}}}}}'
```

If `{unique name}` is supposed to signal to the reader to fill that in themselves, then use `<unique_name>` there.